index.bs

```diff
@@ -1884,18 +1942,26 @@ New Charters and Major Changes</h4>
 	(e.g., regarding deliverables or resource allocation)
 	and include rationale for the changes.
 
+	The Call for Review of a new or substantively modified charter
+	<em class="rfc2119">must</em> include a disposition of comments for those received during the [=charter refinement=] process,
+	highlighting any issues that were closed over sustained objections.
```

Comment:
It's not clear to me what "closed over" means here. Does it mean highlighting any issues that received sustained objections?

Reply:
```suggestion
	highlighting any issues that were closed despite sustained objections.
```